### PR TITLE
Fix tus loading if multiple apache processes are used

### PR DESCRIPTION
**Pull request by azhavoro**
### How has this been tested?

Manually by set `--processes 3` for `mod_wsgi` settings

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into the [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Comment by nmanovic** [comment]
@azhavoro , need to add a line into CHANGELOG. Do we have a test for TUS upload feature?

File: cvat/apps/engine/mixins.py [hunk 1]
```diff
@@ -1,33 +1,91 @@
...
+                data = json.load(fp)
+            return TusFile.TusMeta(**data)
+
+        def save(self):
+            if self._meta is not None:
+                os.makedirs(os.path.dirname(self._path), exist_ok=True)
+                with open(self._path, "w") as fp:
+                    json.dump(asdict(self._meta), fp)
+
+        def is_exist(self):
```
Comment (klakhov) [inline cvat/apps/engine/mixins.py:56]: I think calling this method `exists` is better

File: cvat/apps/engine/mixins.py [hunk 2]
```diff
@@ -40,50 +98,42 @@ def write_chunk(self, chunk):
...
+        self.meta_file.delete()
 
     @staticmethod
     def create_file(metadata, file_size, upload_dir):
         file_id = str(uuid.uuid4())
         filename = metadata.get("filename")
-        cache.add("tus-uploads/{}/filename".format(file_id), "{}".format(filename), TusFile._tus_cache_timeout)
-        cache.add("tus-uploads/{}/file_size".format(file_id), file_size, TusFile._tus_cache_timeout)
-        cache.add("tus-uploads/{}/offset".format(file_id), 0, TusFile._tus_cache_timeout)
-        cache.add("tus-uploads/{}/metadata".format(file_id), metadata, TusFile._tus_cache_timeout)
```
Comment (klakhov) [inline cvat/apps/engine/mixins.py:83]: Tus supports custom metadata which can be provided in HTTP_UPLOAD_METADATA header, its supported(but not used in UI). Here we saved it in cache.
Now as I see it wont be written in TusMeta file, do you think we should support each field individually?

**Review by klakhov — COMMENTED** [review]
I see one failed test, but it looks like its unstable test. It failed in one of my pr's too.

File: cvat/apps/engine/mixins.py (hunk 2 above)
Comment (azhavoro) [reply cvat/apps/engine/mixins.py:83]: I haven't seen the `metadata' from the cache used anywhere in the server code, so I removed it,
but we can keep it inside a file for the future

Comment (klakhov) [reply cvat/apps/engine/mixins.py:83]: I think it would be good for the sake of supporting tus protocol

**Review by klakhov — APPROVED** [review]

**Pull request merged**